5/5.6/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM gcr.io/google-appengine/debian8
 
+# Get the list of packages from the base image.
+RUN dpkg-query -W '-f=${binary:Package}\n' > /usr/local/src/packages.list
```

Comment:
I feel that we should create all temporary files into `/tmp` (we can use `mktemp` function) directory. This will help us to find out that the file is temporary while reading the code.